MapboxCoreNavigation.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "MapboxCoreNavigation"
-  s.version = '2.6.0-alpha.2'
+  s.version = '2.6.0-beta.1'
```

Comment:
Was this modified manually? Shouldn't we do Navigation SDK related updates in scope of https://github.com/mapbox/mapbox-navigation-ios/pull/3936?